docs/src/components/landing/DemoVideo.js

```diff
@@ -139,13 +139,21 @@ export default function DemoVideo() {
   const [isMuted, setIsMuted] = React.useState(false);
 
   const handlePlay = () => {
+    if (typeof window !== 'undefined' && window.gtag) {
+      window.gtag('event', 'toolpad_video', {
```

Comment:
I remember from the grooming we would be mimicking youtube events?

Reply:
Do you mean using labels that are similar to the ones used by YouTube?

Reply:
I thought it could be useful, in case we ever decide to switch we don't lose the data. Not super important though